app/src/main/java/org/sopt/sample/base/BindingFragment.kt

```diff
@@ -0,0 +1,32 @@
+package org.sopt.sample.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.fragment.app.Fragment
+import androidx.viewbinding.ViewBinding
+
+// Fragment의 기본을 작성, 뷰 바인딩 활용
+abstract class BindingFragment<B : ViewBinding>(
+    private val bind: (View) -> B,
+    @LayoutRes layoutResId: Int
+) : Fragment(layoutResId) {
+    private var _binding: B? = null
+    protected val binding
+        get() = requireNotNull(_binding!!)
```

Comment:
```suggestion
    protected val binding
        get() = requireNotNull(_binding!!) { "${this::class.java.simpleName} error." }
```
예외 처리를 추가해도 좋을 것 같아요 ~!